lib/openstudio/extension/runner.rb

```diff
@@ -656,8 +656,15 @@ def run_osw(in_osw, run_dir)
           puts 'simulations are not performed, since to the @options[:run_simulations] is set to false'
         end
 
-        # DLM: this does not always return false for failed CLI runs, consider checking for failed.job file as backup test
+        # Additional checks for failed CLI
+        if File.exist?(File.join(run_dir, 'failed.job'))
+          result = false
+        end
 
+        if !File.exist?(File.join(run_dir, 'finished.job'))
+          result = false
+        end
+        
```

Comment:
what happens if both a `failed.job` and `finished.job` exist? The code here will still return false.

Reply:
yeah but that makes sense to me.

Reply:
okay. looks good. i approve 👍  😜